WeatherForecast/WeatherForecast/Data/Services/ImageChacheService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ImageChacheService.swift
+//  WeatherForecast
+//
+//  Created by JunHeeJo on 2/6/24.
+//
+
+import Foundation
+
+protocol ImageChacheService {
```

Comment:
p3;

```suggestion
protocol ImageCacheService {
```
오타가 있어 보입니다 👀